lib/game/client/game-engine.js

```diff
@@ -169,6 +192,40 @@ class GameEngine extends Game
         this.eventsManager.emit('reldens.gameEngineTabTarget', this, closerTarget, previousTarget);
     }
 
+    getScene(scene)
+    {
+        return this.engineDriver.getScene(scene);
+    }
+
+    addScene(key, sceneConfig, autoStart)
```

Comment:
these on the other hand make more sense even when they only run in a line, because for another driver could not be the case, these will be like the "main methods" I was reffering that we "need" on our drivers because these refers to specific functionality